cassandra/src/main/java/org/hibernate/ogm/datastore/cassandra/CassandraDialect.java

```diff
@@ -210,12 +236,61 @@ public void removeTuple(EntityKey key, TupleContext tupleContext) {
 
 	@Override
 	public Association getAssociation(AssociationKey key, AssociationContext associationContext) {
-		return null; // TODO
+
+		String table = key.getTable();
+		Table tableMetadata = provider.getMetaDataCache().get( key.getTable() );
```

Comment:
Ok, I thought I've read somewhere that `HashMap#get()` may actually mutate some internal state, but I can't find that reference right now.

I guess I'd sleep better if was wrapped in `Collections#unmodfiableMap`.